lib/virt_autotest/common.pm

```diff
@@ -333,7 +333,7 @@ if (get_var("REGRESSION", '') =~ /xen/) {
     } elsif (is_sle('>=16')) {
         # SLES16+ support - only EFI guests are supported, no BIOS guests
         # Both online and full installation types supported for SLES16 EFI guests
-        my @allowed_guests = qw(sles15sp7 sles15sp7efi sles16efi_online sles16efi_full);
+        my @allowed_guests = qw(sles16efi_online sles16efi_full);
```

Comment:
Hi @RoyCai7 ,
thanks for the fix! I think we need to test sle15sp7 guests on sle16 host. Why excluding them?

Reply:
sles15sp7 have autoyast xml issue, we can separate them, and then fix it again.

Reply:
what's your opinion about it?
https://progress.opensuse.org/issues/193030
it blocked the job's success.

Reply:
> yesterday, I basically failed all sles15sp7 testing due to xml issue, it wasted a lot of debugging time.
I think I can fix it after this merge quickly.

I see, if this is the case, fine to exclude temporarily.
So does https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/24085#discussion_r2558303534. It is fine to update in new PR later. Then we can have the fix for full repo link and secure boot immediately. Deal?

Reply:
yes, I think so.